targets/csharp/source/PlayFabSDK/source/PluginManager.cs

```diff
@@ -56,6 +56,9 @@ private IPlayFabPlugin GetPluginInternal(PluginContract contract, string instanc
                     case PluginContract.PlayFab_Transport:
                         plugin = this.CreatePlugin<PlayFabSysHttp>();
                         break;
+                    case PluginContract.PlayFab_TransportWithPolly:
```

Comment:
Building on the other comments. For now, I would prefer if PluginManager didn't have any special knowledge of this plugin. Instead, games could call SetPlugin after creating an instance of PlayFabPollyHttp to override their transport.

Reply:
Sounds good, will adjust. I'll reping you once this has been dogfooded